src/listfunc.c

```diff
@@ -1729,6 +1732,64 @@ Obj FuncCOPY_LIST_ENTRIES( Obj self, Obj args )
 
 }
 
+
+Obj FuncLIST_WITH_IDENTICAL_ENTRIES(Obj self, Obj n, Obj obj)
+{
+    if (!IS_NONNEG_INTOBJ(n)) {
+        ErrorQuit("<n> must be a non-negative integer (not a %s)",
+                  (Int)TNAM_OBJ(n), 0L);
+    }
+
+    Obj  list = 0;
+    Int  len = INT_INTOBJ(n);
+    UInt tnum = TNUM_OBJ(obj);
+
+    if (tnum == T_CHAR) {
+        UChar c = *(UChar *)ADDR_OBJ(obj);
+        list = NEW_STRING(len);
+        memset(CHARS_STRING(list), c, len);
+    }
+    else if (obj == True || obj == False) {
+        list = NewBag(T_BLIST, SIZE_PLEN_BLIST(len));
+        ADDR_OBJ(list)[0] = n;
```

Comment:
Huh, wait, shouldn't that be `= len`? (entirely my fault, of course)

Perhaps also switch to `SET_LEN_BLIST(list, len);`

Reply:
This suggests we should have (more) tests for this